boa/examples/classes.rs

```diff
@@ -0,0 +1,84 @@
+use boa::builtins::object::{Class, ClassBuilder};
+use boa::builtins::value::*;
+use boa::exec::*;
+use boa::realm::Realm;
+use boa::*;
```

Comment:
I don't think we should use `*` imports. Also, I think it is easier to understand if we indent it:

```suggestion
use boa::{
    builtins::{
        value::*,
        object::{Class, ClassBuilder}
    },
    exec::*,
    realm::Realm,
    *,
};
```

Reply:
> I don't think we should use `*` imports. Also, I think it is easier to understand if we indent it:

This is where the `prelude` #440 would be really nice.